schema/record-syntax.ttl

```diff
@@ -29,46 +30,69 @@ skos:relatedProperty rdf:type owl:AnnotationProperty .
+              rdfs:comment "This relates a record to all the objects described in its member graphs. All IRIs described must occur in some of the content, and for any IRI occurring in the content/dataset, there must be a path between that IRI and one of the describes IRIs using only triples occurring in the content/dataset." ;
               rdfs:label "describes" ;
               rdfs:seeAlso <http://purl.obolibrary.org/obo/IAO_0000219> .
 
 
+###  https://rdf.equinor.com/ontology/record/hasMember
+rec:hasMember rdf:type owl:ObjectProperty ;
+              rdfs:subPropertyOf prov:hadMember ;
+              rdfs:domain rec:Record ;
+              rdfs:comment "A relation between some record and some other named graph. This means that the metadata available in the record is applicable to the member graph." ;
+              rdfs:label "has member" .
+
+
 ###  https://rdf.equinor.com/ontology/record/isInScope
 rec:isInScope rdf:type owl:ObjectProperty ;
               rdfs:domain rec:Record ;
-              rdfs:comment "Declares the scope in which the content of the record is valid. When there are several triples with the same subject the meaning is that the content is valid in the intersection of all the scopes. The scopes can be used to route the record to the correct storage and to regulate access. This relation propagates along the inverse of rec:isSubRecordOf" ;
+              rdfs:comment "Declares the scope in which the content of the record is valid. When there are several triples with the same subject the meaning is that the content is valid in the intersection of all the scopes. The scopes can be used to route the record to the correct storage and to regulate access. This relation propagates along rec:hasMember" ;
               rdfs:label "is in scope" ;
               rdfs:seeAlso <http://purl.obolibrary.org/obo/IAO_0000142> .
 
 
-###  https://rdf.equinor.com/ontology/record/isSubRecordOf
-rec:isSubRecordOf rdf:type owl:ObjectProperty ;
-                  rdfs:subPropertyOf [ owl:inverseOf prov:hadMember
-                                     ] ;
-                  rdfs:domain rec:Record ;
-                  rdfs:range rec:Record ;
-                  rdfs:comment "A record can be a sub-record of another record. This does _not_ mean that the contents are related. Scopes are inherited from superrecord to subrecord, and subrecords are inherited via the replaces relation." .
+###  https://rdf.equinor.com/ontology/record/related
+rec:related rdf:type owl:ObjectProperty ;
+            rdfs:subPropertyOf skos:related ;
+            rdfs:domain rec:Record ;
+            rdfs:comment "A relation between a record and some IRI. These IRIs are related to the record in some undefined manner. The IRIs are used as a type of soft scoping." ;
```

Comment:
```suggestion
            rdfs:comment "A relation between a record and some IRI. These IRIs are related to the content of the record in some undefined manner. This can be used as a less strict version of rec:isInScope, f.e.x when indexing is needed but branching is not wanted." ;
```